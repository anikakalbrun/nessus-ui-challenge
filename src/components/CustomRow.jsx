import React from 'react';
import {node} from 'prop-types';

CustomRow.propTypes = {
    children: node
};

export default function CustomRow({children}) {
    return (
        <div className='p-10'>
            <div className='row'>
                <div className='col-xs-12 col-sm-10 col-md-8 col-sm-offset-1 col-sm-offset-2 col-lg-offset-2'>
                    {children}
                </div>
            </div>
        </div>
    )
}



