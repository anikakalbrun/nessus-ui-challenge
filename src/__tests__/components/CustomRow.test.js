import React from 'react';
import CustomRow from '../../components/CustomRow';

let wrapper;

describe('CustomRow', () => {
    it('renders correctly', () => {
        wrapper = renderer.create(<CustomRow/>).toJSON();
        expect(wrapper).toMatchSnapshot();
    });

    it('renders its children', () => {
        wrapper = shallow(<CustomRow children={<p/>}/>);
        expect(wrapper.find('p').length).toEqual(1);
    });
});
