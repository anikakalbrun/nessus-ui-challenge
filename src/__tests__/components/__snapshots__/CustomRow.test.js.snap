// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomRow renders correctly 1`] = `
<div
  className="p-10"
>
  <div
    className="row"
  >
    <div
      className="col-xs-12 col-sm-10 col-md-8 col-sm-offset-1 col-sm-offset-2 col-lg-offset-2"
    />
  </div>
</div>
`;
